cmd/tk/workflow.go

```diff
@@ -38,12 +40,16 @@ func applyCmd() *cobra.Command {
 }
 
 func diffCmd() *cobra.Command {
+	// completion
+	cmp.Handlers.Add("diffStrategy", complete.PredictSet("native", "subset"))
```

Comment:
For extra marks you could automatically detect the client and server version, and "do the right thing":

```
$ kubectl version -o json
{
  "clientVersion": {
    "major": "1",
    "minor": "15",
    "gitVersion": "v1.15.2",
    "gitCommit": "f6278300bebbb750328ac16ee6dd3aa7d3549568",
    "gitTreeState": "clean",
    "buildDate": "2019-08-05T16:54:35Z",
    "goVersion": "go1.12.7",
    "compiler": "gc",
    "platform": "darwin/amd64"
  },
  "serverVersion": {
    "major": "1",
    "minor": "11+",
    "gitVersion": "v1.11.10-gke.5",
    "gitCommit": "5aa3a95d828fe45aab3611dfc4ebdc0341fe1507",
    "gitTreeState": "clean",
    "buildDate": "2019-05-29T17:25:39Z",
    "goVersion": "go1.10.8b4",
    "compiler": "gc",
    "platform": "linux/amd64"
  }
}
```

Reply:
I do! If the flag is unset, the right one is chosen

Reply:
https://github.com/sh0rez/tanka/blob/ef86144866cede78fd799175eaef922539288872/pkg/kubernetes/kubernetes.go#L87-L94

Reply:
Oh nice.